minitwit/src/main/java/SqlDatabase.java

```diff
@@ -1,22 +1,27 @@
-import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URISyntaxException;
 import java.sql.Connection;
 import java.sql.DriverManager;
-import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
 
-public class SQLite {
+public class SqlDatabase {
+
     private Connection connection = null;
 
     public Connection connect() throws SQLException {
-        var file = System.getenv("MINITWIT_DB_PATH");
-        if (file == null) {
-            file = "minitwit.db";
+        String password = System.getenv("MINITWIT_DB_PASS");
+
+        if (password == null) {
+            throw new IllegalStateException("Password for the database is required");
         }
-        connection = DriverManager.getConnection("jdbc:sqlite:" + file);
+
+        connection = DriverManager.getConnection(
+                "jdbc:postgresql://164.90.197.133:5432/minitwit",
+                "minitwituser", password
```

Comment:
Can we also make the connection string and username environment variables? Makes it easier to switch out DB in case IP changes later.

Reply:
Not sure, maybe nice to have the option later :)